docs/source/testing_services.rst

```diff
@@ -0,0 +1,205 @@
+.. _testing_services:
+
+================
+Testing services
```

Comment:
Service in our terminology  is a proper noun so should be capitalised

Reply:
Isn't it a common noun? Also I've written all the titles and subtitles in the `octue` documentation in sentence case - i.e. the first word is capitalised and the rest aren't

Reply:
The way I've been using terms like services is to capitalise them when they're defined and then when instances of them are referred to in singular or plural to lowercase them (I guess like how python classes are capitalised and instances are lowercased. Is this not correct?